docs/content/troubleshooting/troubleshoot-ingress.md

```diff
@@ -0,0 +1,12 @@
+Checking the Events of an Ingress Resource
+After you create or update an Ingress resource, you can immediately check if the NGINX configuration for that Ingress resource was successfully applied by NGINX:
+
+$ kubectl describe ing cafe-ingress
```

Comment:
```suggestion
`kubectl describe ing cafe-ingress`
```

Reply:
@jasonwilliams14 I'm making some small edit suggestions for formatting as I go along; this looks invalid. Should the 'ing' be there?

Reply:
@ADubhlaoich oo, yea...the `ing` is short for `ingress` in the kubernetes world. :)